packages/blocks/stopwatch/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "stopwatch",
+  "version": "0.0.0",
+  "description": "timer block",
+  "keywords": [
+    "blockprotocol",
+    "blocks"
+  ],
+  "license": "MIT",
+  "author": "Tim Diekmann",
+  "scripts": {
+    "build": "block-scripts build",
+    "dev": "block-scripts dev",
+    "fix:eslint": "eslint --ext .ts,.tsx --fix ./src/",
+    "lint:eslint": "eslint --ext .ts,.tsx ./src/",
+    "lint:tsc": "tsc --noEmit",
+    "serve": "block-scripts serve"
+  },
+  "dependencies": {
+    "blockprotocol": "0.0.7"
+  },
+  "devDependencies": {
+    "block-scripts": "0.0.5",
+    "mock-block-dock": "0.0.7",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "block-scripts": {
+    "port": 63212
```

Comment:
Might be worth picking a port that does not conflict with other blocks

Reply:
Changed the port to `63214` in b750a4022aad3548898178772a250548e2b4c354